code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm

```diff
@@ -423,8 +429,7 @@
 	add_cooldown()
 	receiver.gain_plasma(300)
 	succeed_activate()
-	to_chat(owner, span_xenonotice("We transfer some plasma to [target]."))
-	to_chat(receiver, span_xenonotice("We feel our plasma reserves increase. Bless the Queen!"))
+	receiver.balloon_alert_to_viewers("Plasma given from Queen")
```

Comment:
Debatable if it should be visible to all but ill leave it up to you

Reply:
A sudden jump in plasma and also reminds everyone nearby about the (frankly overpowered) ability of the queen to grant large amounts of plasma.

This decision to make this message visible for everyone is coming more from my preference as a queen main. Not seeing transfer plasma used by a queen is very concerning and is indicative of an inexperienced player but detecting its usage is difficult at best.

The message now being visible will allow the more experienced players to remind the queen to occasionally use their abilities to help those around them.